src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,7 @@
-const booleanToWord = (boolean) => {};
-
+const booleanToWord = (boolean) => {
+    if (boolean) {
+        return "Yes";
+    }
+    return "No";
+};
```

Comment:
This should use 2 spaces for indentation. Also, to make the function more secure you could add a check the input value is actually the boolean value of `true` and not just some other truthy value like a string:
```
const booleanToWord = (boolean) => {
  if (boolean === true) {
    return "Yes";
  }
  return "No";
};

```